Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -571,7 +571,7 @@ extension FirefoxHomeViewController: DataObserverDelegate {
 
     func getTopSites() -> Success {
         let numRows = max(self.profile.prefs.intForKey(PrefsKeys.NumberOfTopSiteRows) ?? TopSitesRowCountSettingsController.defaultNumberOfRows, 1)
-        let maxItems = UIDevice.current.userInterfaceIdiom == .pad ? 32 : 16
+        let maxItems = UIDevice.current.isPad ? 32 : 16
```

Comment:
I suggest replacing this with a UITraitCollection based approach. You could be on an iPad, but in split screen mode.